filesystem-server.js

```diff
@@ -0,0 +1,705 @@
+                        name: basename(blob.name),
+                        size: blob.properties.contentLength,
+                        type: extname(blob.name)
+                    }
+                    totalFiles.push(fileData);
+                    const blobClient = containerClient.getBlobClient(blob.name);
+                    await blobClient.delete();
+                }
+            }
+        }
+
+        let response = { cwd: null, details: null, error: null, files: totalFiles };
+        response = JSON.stringify(response);
+        res.setHeader('Content-Type', 'application/json');
+        res.json(response);
+    }
+    catch (error) {
+        var errorMsg = new Error();
+        errorMsg.message = "file not found in given location.";
+        errorMsg.code = "404";
+        res.statusMessage = "File not found in given location.";
+        response = { cwd: null, files: null, details: null, error: errorMsg };
+        res.setHeader('Content-Type', 'application/json');
+        res.json(response)
+    }
+}
+
+async function getDetails(req, res) {
+    try {
+        //For empty details
+        if (req.body.names.length == 0 && req.body.data != 0) {
+            let lastUpdated = null;
+            //Get the folder name from the data
+            req.body.names = req.body.data.map(item => item.name);
+            let size = 0;
+            for await (const blob of containerClient.listBlobsFlat({ prefix: directoryName + req.body.path })) {
+                size += blob.properties.contentLength;
+                if (lastUpdated === null || lastUpdated < blob.properties.lastModified) {
+                    lastUpdated = blob.properties.lastModified;
+                }
+            }
+            const fileDetails = {
+                name: req.body.names[0],
+                location: directoryName + req.body.path,
+                isFile: false,
+                size: await byteConversion(size),
+                created: null,
+                modified: lastUpdated,
+                multipleFiles: false
+            }
+            let response = {};
+            response = { cwd: null, files: null, error: null, details: fileDetails };
+            response = JSON.stringify(response);
+            res.setHeader('Content-Type', 'application/json');
+            res.json(response)
+        } else {
+            let fileDetails = {};
+            let size = 0;
+            let names = [];
+            let location;
+            let isFile = false;
+            let created;
+            let modified;
+            for (const item of req.body.names) {
+                if (req.body.data[0].isFile) {
+                    const blobClient = containerClient.getBlobClient(directoryName + req.body.path + item);
+                    const properties = await blobClient.getProperties();
+                    names.push(basename(blobClient.name));
+                    // Replace the blobClient.name to get the common loaction for more thatn one files
```

Comment:
Correct the spelling in comment lines @vishwanathan3797

Reply:
corrected the spelling